bikeshare_app/views.py

```diff
@@ -107,6 +107,13 @@ def register():
         # Allow user through if check passed, else log attempt and return home
         return render_template('register.html')
 
+@app.route('/checkin', methods=['GET'])
```

Comment:
Should I remove the logic underneath the function 
`    if request.method == 'GET':
        # Allow user through if check passed, else log attempt and return home
        return render_template('checkin.html')`
and instead just return the template? 
Also, should I remove the the `methods = ['GET]` parameter of the function.